homeassistant/components/tado/device_tracker.py

```diff
@@ -40,15 +41,28 @@ async def async_get_scanner(
 ) -> DeviceScanner | None:
     """Configure the Tado device scanner."""
     device_config = config["device_tracker"]
-    import_result = await hass.config_entries.flow.async_init(
-        DOMAIN,
-        context={"source": SOURCE_IMPORT},
-        data={
-            CONF_USERNAME: device_config[CONF_USERNAME],
-            CONF_PASSWORD: device_config[CONF_PASSWORD],
-            CONF_HOME_ID: device_config.get(CONF_HOME_ID),
-        },
-    )
+    try:
+        import_result = await hass.config_entries.flow.async_init(
+            DOMAIN,
+            context={"source": SOURCE_IMPORT},
+            data={
+                CONF_USERNAME: device_config[CONF_USERNAME],
+                CONF_PASSWORD: device_config[CONF_PASSWORD],
+                CONF_HOME_ID: device_config.get(CONF_HOME_ID),
+            },
+        )
+    except PyTado.exceptions.TadoWrongCredentialsException as err:
+        _LOGGER.error("Error importing Tado integration: %s", err)
+        async_create_issue(
+            hass,
+            DOMAIN,
+            "failed_import_invalid_auth",
+            breaks_in_ha_version="2024.7.0",
```

Comment:
The other version in this file is 2024.6.0, which one is correct?

Reply:
Should be the half year retention, but it all depends on when it can be merged, right?

Reply:
I mean this is a bug iirc, so it would get into 2024.1.0

Reply:
Agree we should fix this before 2024.1.0 as this is going to be a problem for users being migrated.  Add the tag. We can always push it if its not ready in time but it sure would be nice to ship with this fixed

Reply:
I think all of them should be 2024.7.0 since we release it in January. January + 6 is July.